USERS.md

```diff
@@ -509,6 +509,12 @@ Users (Alphabetically)
       U: CNI (instead of amazon-vpc-cni-k8s), DefaultDeny(Zero Trust), Hubble, CiliumNetworkPolicy, CiliumClusterwideNetworkPolicy, EKS
       L: https://schubergphilis.com/en
       Q: @stimmerman @shoekstra @mbaumann
+      
+    * N: SIGHUP
```

Comment:
Why this change? Seems to be copied from the block below. :thinking:

Reply:
Hum I made a mistake somewhere during my rebasing... I will fix that